dpkg/names.py

```diff
@@ -0,0 +1,10 @@
+OBJECTS_TABLE_NAME = "objects_table"
+LINKS_TABLE_NAME = "links_table"
+TRACKS_TABLE_NAME = "tracks_table"
+
+X_COORD_NAME = "x_coord_cmso"
+Y_COORD_NAME = "y_coord_cmso"
+Z_COORD_NAME = "z_coord_cmso"
+FRAME_NAME = "frame_cmso"
+OBJECT_NAME = "object_id_cmso"
+LINK_NAME = "link_id_cmso"
```

Comment:
This file (does not necessarily have to be called `names.py`, we can change that here or in the future) is meant to be the single source where our resource / column names are defined. Everywhere else in the code we should use these vars (e.g., `names.OBJECT_NAME`) rather than hardwire the values. I have kept the current values (e.g., `"object_id_cmso"`), but they can change depending on what we agree upon for the spec (keeping in mind datapackage spec compliance, of course). Indeed, I have proposed some changes myself in the PR description.